edb/schema/annos.py

```diff
@@ -31,6 +31,11 @@
 from . import utils
 
 
+if typing.TYPE_CHECKING:
```

Comment:
`from typing import *  # NoQA`?

Reply:
I have a script to refactor all those at a later point. Same what I used for object to typing.Any yesterday. I can apply that in this pull request in the last commit. For now I'd like the clarity of which Tuple is which, and so on.

Reply:
Sure!

Reply:
Sure!